essentials-framework/Essentials Devices Common/Essentials Devices Common/VideoCodec/CiscoCodec/CiscoSparkCodec.cs

```diff
@@ -15,16 +15,17 @@
 using PepperDash.Essentials.Core.Routing;
 using PepperDash.Essentials.Devices.Common.Cameras;
 using PepperDash.Essentials.Devices.Common.Codec;
-using PepperDash.Essentials.Core;
 using PepperDash.Essentials.Devices.Common.VideoCodec;
 
 namespace PepperDash.Essentials.Devices.Common.VideoCodec.Cisco
 {
     enum eCommandType { SessionStart, SessionEnd, Command, GetStatus, GetConfiguration };
+	public enum eExternalSourceType {camera, desktop, document_camera, mediaplayer, PC, whiteboard, other}
+	public enum eExternalSourceMode {Ready, NotReady, Hiddon, Error} 

```

Comment:
It's a small thing...but let's make `Hiddon` be `Hidden`

Reply:
Noted, corrected.